CHANGELOG.rst

```diff
@@ -0,0 +1,281 @@
+- iam_role: The default value of the purge_policies has been deprecated and will change from true to false in Ansible 2.14
+- s3_lifecycle - The ``requester_pays`` option had no effect and will be removed in Ansible 2.14
+- s3_sync - The ``retries`` option had no effect and will be removed in Ansible 2.14
+
+Bugfixes
+--------
+
+- **security issue** - Convert CLI provided passwords to text initially, to prevent unsafe context being lost when converting from bytes->text during post processing of PlayContext. This prevents CLI provided passwords from being incorrectly templated (CVE-2019-14856)
+- **security issue** - Update ``AnsibleUnsafeText`` and ``AnsibleUnsafeBytes`` to maintain unsafe context by overriding ``.encode`` and ``.decode``. This prevents future issues with ``to_text``, ``to_bytes``, or ``to_native`` removing the unsafe wrapper when converting between string types (CVE-2019-14856)
+- azure_rm_dnsrecordset_info - no longer returns empty ``azure_dnsrecordset`` facts when called as ``_info`` module.
+- azure_rm_resourcegroup_info - no longer returns ``azure_resourcegroups`` facts when called as ``_info`` module.
+- azure_rm_storageaccount_info - no longer returns empty ``azure_storageaccounts`` facts when called as ``_info`` module.
+- azure_rm_virtualmachineimage_info - no longer returns empty ``azure_vmimages`` facts when called as ``_info`` module.
+- azure_rm_virtualmachinescaleset_info - fix wrongly empty result, or ``ansible_facts`` result, when called as ``_info`` module.
+- azure_rm_virtualnetwork_info - no longer returns empty ``azure_virtualnetworks`` facts when called as ``_info`` module.
+- cloudfront_distribution - Always add field_level_encryption_id to cache behaviour to match AWS requirements
+- cloudwatchlogs_log_group - Fix a KeyError when updating a log group that does not have a retention period (https://github.com/ansible/ansible/issues/47945)
+- cloudwatchlogs_log_group_info - remove limitation of max 50 results
+- ec2_asg: Ensure "wait" is honored during replace operations
+- ec2_launch_template - Update output to include latest_version and default_version, matching the documentation
+- ec2_transit_gateway - Use AWSRetry before ClientError is handled when describing transit gateways
+- ec2_transit_gateway - fixed issue where auto_attach set to yes was not being honored (https://github.com/ansible/ansible/issues/61907)
+- edgeos_config - fix issue where module would silently filter out encrypted passwords
+- fixed issue with sns_topic's delivery_policy option resulting in changed always being true
+- lineinfile - properly handle inserting a line when backrefs are enabled and the line already exists in the file (https://github.com/ansible/ansible/issues/63756)
+- route53 - improve handling of octal encoded characters
+- win_credential - Fix issue that errors when trying to add a ``name`` with wildcards.
+
+New Modules
```

Comment:
The New Modules list looks rather long...

Reply:
This is the 1.0.0 section, when all modules were added to the collection.